data/raw/USB6014_core_nodes.txt

```diff
@@ -0,0 +1,14 @@
+51
```

Comment:
Same with this script, maybe we have a README in the raw data directory for now.